File: Barista.Coordinator/Repository/Coffee.cs
using System;

namespace Barista.Coordinator.Repository
{
  public class Coffee
  {
    public Guid Id { get; set; }
    public Guid OriginalOrderId { get; set; }
    public string Type { get; set; }
    public int NumberOfSugars { get; set; }
    public int NumberOfCreamers { get; set; }
    public bool IsComplete { get; set; }
  }
}

File: Barista.Web/js/Coffee.js
let $ = require('jquery');

function Coffee(model) {
  let self = this;

  self.id = model.id;
  self.orderId = model.originalOrderId,
  self.type = model.type;
  self.sugars = model.numberOfSugars;
  self.creamers = model.numberOfCreamers;
}

module.exports = Coffee;